pom.xml

```diff
@@ -136,6 +136,11 @@
 			<artifactId>junit</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.venice.piazza</groupId>
+			<artifactId>pz-jobcommon</artifactId>
+			<version>2-0-0-7-gb51fe13</version>
```

Comment:
Is this version pegging going to be problematic for maintainability?

Reply:
This is how all of our libraries are versioned (apps, too, for that matter). Updates contain potentially breaking changes, so we do not want to automate with some LATEST version -- if that's what you mean.